osgp/platform/osgp-adapter-kafka-logging/src/main/java/org/opensmartgridplatform/adapter/kafka/logging/config/GridMeasurementPublishedEventKafkaConsumerConfig.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.kafka.logging.config;
+
+import org.opensmartgridplatform.adapter.kafka.da.avro.GridMeasurementPublishedEvent;
+import org.opensmartgridplatform.shared.application.config.kafka.AbstractKafkaConsumerConfig;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Conditional;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory;
+import org.springframework.kafka.core.ConsumerFactory;
+
+@Configuration
+@Conditional(PeakShavingLoggingEnabled.class)
```

Comment:
Should probably be a conditional on the `GridMeasurementPublishedEventLoggingEnabled.class` instead.